src/main/java/ch/njol/skript/expressions/ExprExplodedBlocks.java

```diff
@@ -68,6 +85,45 @@ protected Block[] get(Event e) {
 		List<Block> blockList = ((EntityExplodeEvent) e).blockList();
 		return blockList.toArray(new Block[blockList.size()]);
 	}
+
+	@Override
+	public @Nullable Class<?>[] acceptChange(ChangeMode mode) {
+		switch (mode) {
+			case ADD:
+			case REMOVE:
+			case SET:
+				return CollectionUtils.array(Block[].class);
+			case DELETE:
+				return CollectionUtils.array();
```

Comment:
it actually should be okay to return an array with contents for DELETE. i don't believe EffChange cares about the contents for DELETE/RESET, just that it isn't null